cdap-examples/WikipediaPipeline/src/main/java/co/cask/cdap/examples/wikipedia/WikipediaService.java

```diff
@@ -91,6 +93,12 @@ public void getTopics(HttpServiceRequest request, HttpServiceResponder responder
     @Path("/lda/topics/{topic}")
     public void getTopic(HttpServiceRequest request, HttpServiceResponder responder,
                          @PathParam("topic") Integer topic) {
+      String namespace = getContext().getRuntimeArguments().get("namespace");
```

Comment:
but why find out the name again and again shouldn't it be determined in the initialize of the service once for the service ?